driver/razermouse_driver.c

```diff
@@ -2971,6 +3092,16 @@ static void razer_mouse_disconnect(struct hid_device *hdev)
         device_remove_file(&hdev->dev, &dev_attr_device_mode);
 
         switch(usb_dev->descriptor.idProduct) {
+        case USB_DEVICE_ID_RAZER_LANCEHEAD_WIRELESS:
+            device_remove_file(&hdev->dev, &dev_attr_charge_effect);
+            device_remove_file(&hdev->dev, &dev_attr_charge_colour);
+        /* fallthrough */
+        case USB_DEVICE_ID_RAZER_LANCEHEAD_WIRED:
+            device_remove_file(&hdev->dev, &dev_attr_charge_level);
+            device_remove_file(&hdev->dev, &dev_attr_charge_status);
+            device_remove_file(&hdev->dev, &dev_attr_charge_low_threshold);
+            device_remove_file(&hdev->dev, &dev_attr_device_idle_time);
+        /* fallthrough */
```

Comment:
indent the comments one more to the right please, to be in line with the statements before it

Reply:
I made the changes as requested, but then Travis CI complains and the checks fail because this seems to go against the astyle linux formatting...